packages/tic-tac-toe/app/services/user.ts

```diff
@@ -20,8 +20,8 @@ export default class UserService extends Service {
 
   private async requestAddresses(): Promise<void> {
     await window.channelProvider.enable();
-    this.walletAddress = window.channelProvider.signingAddress as string;
-    this.userAddress = window.channelProvider.selectedAddress as string;
+    this.address = window.channelProvider.signingAddress as string;
+    this.outcomeAddress = window.channelProvider.selectedAddress as string;
```

Comment:
Is this better?
`this.address = window.channelProvider.signingAddress?.toString() ?? '';`
Otherwise, I get errors:
```
(property) UserService.address: string
Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.ts(2322)
```

Reply:
I would say it was marginally better, since you are explicitly dealing with the case where `window.channelProvider.signingAddress` is undefined, rather than risking that causing an exception at runtime.  

I don't think you need the `.toString()` bit, though. 

We should reserve the use of `as` for cases where we know better than the TS compiler: e.g. we knew for sure that `window.channelProvider.signingAddress` is a string, but for some reason the TS compiler cannot infer that. I don't think that is the case here.

Reply:
Ok makes sense. I'll do this in my next PR. Thanks!